### PR TITLE
global: bibrecord dependency removal

### LINKED ISSUE #18: utils: remove legacy functions

### PR DESCRIPTION
**Pull request by hachreak**
* INCOMPATIBLE Removes legacy bibrecord dependency.
  (addresses inveniosoftware/invenio#3233) (closes #18)
  (addresses inveniosoftware/invenio#3508)

* fix wheel setup configuration.

### CODE REVIEW

**Comment by hachreak** [comment]
ping @jirikuncar (devel is green)

File: invenio_records/views.py [hunk 1]
```diff
@@ -145,10 +133,6 @@ def record_context():
                visible_when=visible_collection_tabs('metadata'))
 def metadata(recid, of='hd', ot=None):
     """Display formated record metadata."""
-    # from invenio.legacy.bibrank.downloads_similarity import \
```
Comment (jirikuncar) [inline invenio_records/views.py]: please add at least a `# TODO add signal support`

File: invenio_records/views.py [hunk 2]
```diff
@@ -91,18 +91,6 @@ def decorated(recid, *args, **kwargs):
             flash(auth_msg, 'error')
             abort(apache.HTTP_UNAUTHORIZED)
 
-        # from invenio.legacy.search_engine import record_exists, \
```
Comment (jirikuncar) [inline invenio_records/views.py]: add `# TODO check record status (exists, merged, deleted)` + create an issue. thanks

Comment (hachreak) [inline invenio_records/views.py]: done

File: invenio_records/views.py (hunk 1 above)
Comment (hachreak) [inline invenio_records/views.py]: done

**Pull request merged**